client/my-sites/customer-home/cards/tasks/site-setup-list/style.scss

```diff
@@ -65,6 +65,24 @@
 				border-bottom: none;
 			}
 		}
+
+		.site-setup-list__nav-item-email {
+			color: var( --studio-gray-50 );
+			display: block;
```

Comment:
I think on desktop it would be better if the email is not bold:

This PR:
![](https://cldup.com/i0o6jF5VP1.png)

My suggestion:

![](https://cldup.com/41u7jyP4IT.png)